mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -73,15 +74,52 @@ final class Bucketizer @Since("1.4.0") (@Since("1.4.0") override val uid: String
   @Since("1.4.0")
   def setOutputCol(value: String): this.type = set(outputCol, value)
 
+  /**
+   * Param for how to handle invalid entries. Options are skip (which will filter out rows with
+   * invalid values), or error (which will throw an error), or keep (which will keep the invalid
+   * values in certain way). Default behaviour is to report an error for invalid entries.
```

Comment:
I'd just write: ```default: "error"```
Rewording as "report" instead of "throw" could confuse people.